### PR TITLE
Add tests utility

### PR DESCRIPTION
**Pull request by samuelfangjw**
Add test utility to make it easier to add tests in the future.

### CODE REVIEW

File: src/main/java/seedu/address/model/project/DeadlineList.java [hunk 1]
```diff
@@ -35,6 +35,14 @@ public DeadlineList(List<CompletableDeadline> deadlines) {
         return this.deadlines;
     }
 
+    /**
+     * Returns a copy of this {@code DeadLineList}
+     * @return A copy of this {@code DeadlineList}
```
Comment (Eriksen2411) [inline src/main/java/seedu/address/model/project/DeadlineList.java:52]: hey samuel I'm not sure about this, but as I see it from javadoc format of other methods, there should be a blank line between description of method and parameter, return, throw

File: src/main/java/seedu/address/model/task/repeatable/Event.java [hunk 2]
```diff
@@ -71,7 +71,9 @@ public boolean equals(Object other) {
 
         Event otherEvent = (Event) other;
         return otherEvent.getDescription().equals(getDescription())
-                && otherEvent.getIsDone().equals(getIsDone());
```
Comment (Eriksen2411) [inline src/main/java/seedu/address/model/task/repeatable/Event.java:68]: I think this is a complicated boolean expression as code quality says about it. You can choose to keep that or separate them 1 by 1

File: src/test/java/seedu/address/testutil/ProjectBuilder.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    /**
+     * Sets the {@code ParticipantList} of the {@code Project} that we are building.
+     */
+    public ProjectBuilder withName(ParticipantList participantList) {
+        this.participantList = participantList.getCopy();
+        return this;
+    }
+
+
+    public Project build() {
```
Comment (Eriksen2411) [inline src/test/java/seedu/address/testutil/ProjectBuilder.java:86]: should this have a javadoc

File: src/test/java/seedu/address/testutil/ProjectsFolderBuilder.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * A utility class to help with building ProjectsFolder objects.
+ * Example usage: <br>
+ *     {@code ProjectsFolder pf = new ProjectsFolderBuilder().withProject(project).build();}
+ */
+public class ProjectsFolderBuilder {
+
+    private ProjectsFolder projectsFolder;
+
+    public ProjectsFolderBuilder() {
```
Comment (Eriksen2411) [inline src/test/java/seedu/address/testutil/ProjectsFolderBuilder.java:15]: this 2 method missing javadoc also, as for other builder, you put javadoc for constructor also

File: src/test/java/seedu/address/testutil/TodoBuilder.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+    /**
+     * Builds the {@code Todo} object.
+     * @return {@code Todo}.
+     */
+    public Todo build() {
+        return new Todo(description, isDone);
+    }
+
+}
```
Comment (Eriksen2411) [inline src/test/java/seedu/address/testutil/TodoBuilder.java:59]: Ahhh, OK I think that is all I can spot for now. For cases to test, I cannot think of other cases to add.

**Review by Eriksen2411 — CHANGES_REQUESTED** [review]

**Review by vevek — APPROVED** [review]
LGTM!

**Review by Eriksen2411 — APPROVED** [review]
LGTM

**Pull request merged**